parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -298,6 +294,12 @@ if [ -f /var/run/reboot-required ]; then
     aptmarkWALinuxAgent unhold &
   fi
 else
+{{- if RunUnattendedUpgrades}}
+  if [[ -z ${MASTER_NODE} ]]; then
+    systemctl_restart 100 5 30 kubelet
```

Comment:
Why restart?  I would expect that you could just "systemctl start kubelet" which is what will happen at reboot time anyway.  You have already enabled it.  This way you could just blanket "start" and if it is already running, nothing will happen and if it is not, it will start it.

Less need to check for master or have other complications.

The problem with restart is that if it is running, it will stop the service first.

Reply:
In this particular condition we know that it hasn't started yet, because when we scaffolded kubelet systemd service earlier we explicitly only "enabled" the service in a truthy `RunUnattendedUpgrades` context.

Your comment is true generally, though. We could add a new `systemctl_start` convenience func in `parts/k8s/cloud-init/artifacts/cse_helpers.sh` and audit the code to use that instead of "restart" in all the places where restart could be disruptive.

Reply:
If we had that helper function (and it would be a trivial function) we could then unconditionally make sure the services are started rather than having to check if we are unattended masters or not.

However, this is a great first step.